separator_test.go

```diff
@@ -0,0 +1,49 @@
+package memefish_test
+
+import (
+	"github.com/cloudspannerecosystem/memefish"
+	"github.com/google/go-cmp/cmp"
+	"regexp"
+	"testing"
+)
+
+func TestSeparateRawStatements(t *testing.T) {
+	for _, test := range []struct {
+		desc  string
+		input string
+		errRe *regexp.Regexp
+		want  []string
+	}{
+		{desc: "empty input", input: "", want: []string{""}},
```

Comment:
This spec seems inconsistent with other specs to me.

That is, `SeparateRawStatements(";")` returns `[]{""}`, so I guess that `SeparateRawStatements` omits the last empty statement and `SeparateRawStatements("")` returns `[]{}`, but this spec says it is NO.

At least, we should note this behavior in the doc.

Reply:
OK. I will write semantics to the go doc comment.

TL;DR

Statements are terminated by `;`, `<eof>`, or `;<eof>` . So, empty input is interpret as a single empty statement.